### DISCUSSION TITLE
does HelmChartConfig file need to present on all server node?

### DISCUSSION THREAD

**Discussion by ibrother** [rancher/rke2#1614] [Q&A] [answered]
hi all

i use a HelmChartConfig file `/var/lib/rancher/rke2/server/manifests/rke2-cilium-config.yaml` to custom cilium's config, for rke2's ha install, does HelmChartConfig file need to present on all server node or just need to present on the first server node?

**Comment by brandond** [comment, marked as answer]
No, it only needs to be present on one of the nodes. After it's been applied to the cluster it does not actually need to continue to be present anywhere; it will remain unless manually deleted.

**Reply by ibrother** [reply]
okay, thanks for your reply.
